### DISCUSSION TITLE
Are you required to attribute GitHub Pages?

### DISCUSSION THREAD

**Discussion by techboyg5** [community/community#22740] [Other Feature Feedback, Questions, & Ideas]
<p>Are you required to attribute GitHub Pages?</p>

**Comment by Jme7** [comment]
<p>What do you mean by attribute?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>HI <a class="mention" href="/techboyg5">@techboyg5</a> thank you for being here! No, you are not required to list pages as your host. The content of Pages is at the user’s discretion, as long as it’s legal and doesn’t otherwise violate the Terms of Service.</p>
